AlertaDengue/AlertaDengue/settings.py

```diff
@@ -155,42 +128,30 @@ def read_admins(value):
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql',
-        'NAME': os.getenv('PSQL_DB'),
-        'USER': os.getenv('PSQL_USER'),
-        'PASSWORD': os.getenv('PSQL_PASSWORD'),
-        'HOST': os.getenv('PSQL_HOST'),
-        'PORT': os.getenv('PSQL_PORT'),
+        'NAME': PSQL_DB,
+        'USER': PSQL_USER,
+        'PASSWORD': PSQL_PASSWORD,
+        'HOST': PSQL_HOST,
+        'PORT': '',
```

Comment:
why it is not using PSQL_PORT?

Reply:
Na configuração anterior a remoção do decouple não havia a variável da porta setada, embora eu tenha criado no arquivo de ambiente .env para os.getenv, tentei manter como estava, (não resultou nenhum erro), mas ainda é possível commitar com a porta setada, sendo que ao topo ainda ficou declarado a variável PSQL_PORT = os.getenv('PSQL_PORT')

Reply:
acredito que seria melhor usar o `PSQL_PORT` no lugar do `''`

Reply:
Ok. 
Thanks for review!